nextflow.config

```
@@ -155,8 +155,11 @@ params {
     krona_taxonomy_directory   = null
 
     // profile standardisation
-    run_profile_standardisation = false
-    generate_biom_output        = false
+    run_profile_standardisation             = false
+    taxpasta_add_taxonomy                   = false
+    taxpasta_add_samplesheet                = false
+    taxpasta_standardisation_format         = 'tsv'
```

```suggestion
    standardisation_taxpasta_format         = 'tsv'
```

Need sto be changed in the pipeline code